src/main/java/beam/analysis/plots/PersonVehicleTransitionStats.java

```diff
@@ -223,7 +229,7 @@ JFreeChart getGraphic(String mode, int iteration) {
         plot.getRangeAxis().setStandardTickUnits(NumberAxis.createIntegerTickUnits());
         final CategoryAxis axis1 = new CategoryAxis("sec");
         axis1.setTickLabelFont(new Font("SansSerif", Font.PLAIN, 7));
-        plot.setDomainAxis(new NumberAxis("time"));
+        plot.setDomainAxis(new NumberAxis("timeBinSize=3600 sec"));
```

Comment:
The _magic string_ `timeBinSize=3600 sec` is repeated two times in this class. Consider extracting a constant with a meaningful name.